cszczepaniak/cribbly-backend
# Fix up player service

# What I added (link any issues addressed)
- Added a team name to the db so we can test out the mapping from team to player
- Remove email from query string and grab it from the header instead for security
- Use a left join so we still return the player if they don't have a team
- Add an endpoint to get player by id

# How to test it
1. Create some players
2. Manually create a team in the db (the table should get created by the migrations, but you'll need to add a record manually)
3. Manually set a player's TeamId to the newly created team's Id
4. Get the player either by Id or Email, see that their team object is populated with a name (and everything else will be null)
5. Get a player who doesn't have a team yet, see that you still get the player but the whole team is null
